### PR TITLE
#1 Homepage

### PR DESCRIPTION
**Pull request by codingknite**

### CODE REVIEW

File: src/components/DiscoverGenres.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import * as BsIcons from "react-icons/bs";
+import * as ImIcons from "react-icons/im";
+
+export default function DiscoverGenres({ data }) {
+  return (
+    <>
+      <div className="display-movies">
```
Comment (idiglove) [inline src/components/DiscoverGenres.jsx:8]: stilll not using styled components yet?

File: src/utils/useFetch.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+export default function useFetch(url) {
+    const isMounted = useRef(false)
+    const [data, setData] = useState(null)
+    const [error, setError] = useState(null)
+    const [loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        isMounted.current = true;
+        async function fetchData() {
```
Comment (idiglove) [inline src/utils/useFetch.js:11]: lol go and try react-query already

**Review by idiglove — CHANGES_REQUESTED** [review]
Looks good, but take a look at my comments

File: src/components/DiscoverGenres.jsx (hunk 1 above)
Comment (codingknite) [reply src/components/DiscoverGenres.jsx:8]: I am...haven't styled the app yet. I just have basic styling just to see where things are but I'm using styled-components

File: src/utils/useFetch.js (hunk 2 above)
Comment (codingknite) [reply src/utils/useFetch.js:11]: Lolol 😁 okay I'm doing it now haha

**Comment by codingknite** [comment]
I added react-query. It'll show up in the next PR

**Pull request merged**